src/index.js

```diff
@@ -144,7 +147,9 @@ export default class StepWizard extends PureComponent {
     }
 
     // Allows for using HTML elements as a step
-    isReactComponent = type => typeof type === 'function'
+    isReactComponent = ({ type }) => (
+        typeof type === 'function' || typeof type === 'object'
```

Comment:
@mathvaleriano I think this should work for you, yeah?

Reply:
yeah... i'm using it for now

Reply:
https://github.com/jcmcneal/react-step-wizard/issues/27#issuecomment-486482549